<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrandInputDialog</class>
 <widget class="QDialog" name="BrandInputDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>691</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Input Brand</string>
  </property>
  <widget class="QPushButton" name="bhUpdateButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>630</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Brand</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="viewButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>101</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>View</string>
   </property>
  </widget>
  <widget class="QPushButton" name="noBrandButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>101</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>No Brand</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>101</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>10</y>
     <width>101</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Alt+X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tagButton">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>330</y>
     <width>101</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Ta&amp;g</string>
   </property>
   <property name="shortcut">
    <string>Alt+G</string>
   </property>
  </widget>
  <widget class="QPushButton" name="headButton">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>250</y>
     <width>101</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Head</string>
   </property>
   <property name="shortcut">
    <string>Alt+H</string>
   </property>
  </widget>
  <widget class="QPushButton" name="positionButton">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>90</y>
     <width>101</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Position</string>
   </property>
   <property name="shortcut">
    <string>Alt+P</string>
   </property>
  </widget>
  <widget class="QPushButton" name="transactionButton">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>170</y>
     <width>101</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Trans.</string>
   </property>
   <property name="shortcut">
    <string>Alt+T</string>
   </property>
  </widget>
  <widget class="QLabel" name="inputLabel">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>410</y>
     <width>190</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>CURRENT EDIT ITEM</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputEdit">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>470</y>
     <width>190</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel3_2">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>170</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Transaction #</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel3">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>90</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Brand Position</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel3_3">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>250</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Head Count</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="currentHeadDisplay">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>280</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel3_4">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>330</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Tag Number</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="currentTagDisplay">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>360</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="currentPositionDisplay">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>120</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="currentTransactionDisplay">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>200</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Q3Frame" name="smallBrandFrame">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>630</y>
     <width>48</width>
     <height>48</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="BrandInputFrame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>571</width>
     <height>581</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3Frame</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">qpen.h</include>
  <include location="local">qpainter.h</include>
  <include location="global">string</include>
  <include location="local">dao/branddao.h</include>
  <include location="local">brandwidget.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
